test/bitcask_pr156.erl

```diff
@@ -135,13 +134,13 @@ check_no_tombstones(Ref, Good) ->
             {check_no_tombstones, Else}
     end.
 
-make_merge_txt(Dir, Seed, Probability) ->
-    random:seed(Seed),
+make_merge_txt(Dir, _Seed, Probability) ->
+    % random:seed(Seed),
     case filelib:is_dir(Dir) of
         true ->
             DataFiles = filelib:wildcard("*.data", Dir),
             {ok, FH} = file:open(Dir ++ "/merge.txt", [write]),
-            [case random:uniform(100) < Probability of
+            [case rand:uniform(100) < Probability of
                  true ->
                      io:format(FH, "~s\n", [DF]);
                  false ->
```

Comment:
@ThomasArts I've stopped using seed here, but I don't know why seed was used and cycled through originally.  The tests pass - but I'm uncomfortable with what I've done.  Can you review please?

Reply:
The seed was hard coded in a test to be {335, 217, 333}, most likely from a given counterexample. 
https://github.com/basho/bitcask/blob/eef50ede6df24b45f6b01b5537da6962eb57c132/test/bitcask_pr156.erl#L49

So indeed, this might be tricky. Trying to run the PULSE tests again.... but we don't know which fault it fixed, otherwise we could re-introduce the fault and see whether we find it with this and/or different values.

Reply:
It is not guaranteed that we would find the bug with the original version of the test, given that the R20 scheduler may be so different. So you're fine, but some more work is needed later to get PULSE up and running in R20 version.